### PR TITLE
Adding test_circle.py to test Circle Class

### LINKED ISSUE #18: Circle Class

### PR DESCRIPTION
**Pull request by ChrisPowza**
Closes Issue #18 

Added tests for the setters and getters for the Circle class, as well as type validation tests for the setters, in test_circle.py.

### CODE REVIEW

**Review by masterninja12225 — APPROVED** [review]

**Review by WMUcaleb — APPROVED** [review]

File: main/tests/test_circle.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+class TestCircle:
+
+    def test_get_coords(self):
+        circle = Circle()
```
Comment (JacobKonkowski) [inline main/tests/test_circle.py:7]: To be able to get coords, you have to initialize a value in the constructor or set the coords with the method. My recommendation is to initialize it with the constructor, so this test isn't dependent on the set_coords method working.

File: main/tests/test_circle.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    def test_set_coords(self):
+        circle = Circle([4, 3, 2, 1, 0])
+        circle.set_coords([0, 1, 2, 3, 4])
+
+        assert circle.get_coords() == [0, 1, 2, 3, 4]
+
+    def test_get_radius(self):
+        circle = Circle()
+        assert circle.get_radius() == 3
+
```
Comment (JacobKonkowski) [inline main/tests/test_circle.py:19]: Similar comment as the one above. I would initialize radius and coordinates in the Circle constructor.

**Review by JacobKonkowski — CHANGES_REQUESTED** [review]

File: main/tests/test_circle.py (hunk 1 above)
Comment (ChrisPowza) [reply main/tests/test_circle.py:7]: got it, I will put in values for the constructor to initialize with

File: main/tests/test_circle.py (hunk 2 above)
Comment (ChrisPowza) [reply main/tests/test_circle.py:19]: same with the other one

**Pull request merged**